resources/utils/config.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+  SUPABASE_URL: process.env.NEXT_PUBLIC_SUPABASE_URL,
```

Comment:
This .env shall be on the readme, and on the secrets the value of them (for staging and production)